mne/viz/epochs.py

```diff
@@ -866,24 +866,21 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
 def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
                     proj=False, bandwidth=None, adaptive=False, low_bias=True,
                     normalization='length', picks=None, ax=None, color='black',
-                    area_mode='std', area_alpha=0.33, dB=True, n_jobs=1,
-                    show=True, verbose=None):
-    """Plot the power spectral density across epochs.
+                    xscale='linear', area_mode='std', area_alpha=0.33,
+                    dB=True, estimate='auto', show=True, n_jobs=1,
+                    average=False, line_alpha=None, spatial_colors=True,
+                    verbose=None):
+    """%(plot_psd_doc)s.
 
     Parameters
     ----------
     epochs : instance of Epochs
         The epochs object
-    fmin : float
-        Start frequency to consider.
-    fmax : float
-        End frequency to consider.
-    tmin : float | None
-        Start time to consider.
-    tmax : float | None
-        End time to consider.
-    proj : bool
-        Apply projection.
+    %(plot_psd_fmin)s
```

Comment:
the idea of using `docdict` is to allow factoring of the documentation in the sense that identical parameters with the same doc are defined only once like this we avoid variated wording for the same concept. 

Stripping them out and giving them a name that is meant to have a single instance defeats entirely this purpose. What I would do is to do this in a different PR and search for all `tmax`, `tmin`, `proj`, etc. and substitute them all at once.

If `fmin` and `fmax` has only meaning in this function, then I would keep them in this function docstring. Same for `plot_psd_doc` its unlikely that this piece of text could be reused anywhere.

Reply:
They are in this case two instances: one for Raw and one for Epochs, but I take your general point that many of these ought to be defined more universally and not as a part of plot_psd. I'm happy to take out the more common ones. It would be worth looking into how common fmin and fmax actually are, and whether it's economical to universalise them as well...